miden-lib/asm/miden/contracts/auth/basic.masm

```diff
@@ -37,14 +37,14 @@ export.auth_tx_rpo_falcon512
     push.PUBLIC_KEY_SLOT exec.account::get_item
     # => [PUB_KEY, M]
 
+    # Update the nonce
+    push.1 exec.account::incr_nonce
+    # => []
```

Comment:
This was needed because otherwise, at the time of requesting the signature, the `AccountDelta` would be invalid due to not having an increased nonce.